### ISSUE TITLE
Support for s390x

### ISSUE THREAD

**Issue by gabor-boros** [srh/rethinkdb-package-builder#5]
Some ubuntu docker images already have an s390x tag, but for those that have no s390x tags, the https://hub.docker.com/r/s390x/ubuntu contains docker images.

So: using the https://hub.docker.com/r/s390x/ubuntu images for s390x builds over the regular ubuntu images would allow having s390x build.

---

Sorry, too much s390x. Also, it is not a priority thing either. All build steps are the same, even the packages, only the base image should be "replaced" in the Dockerfiles.

**Comment by srh** [comment]
It sounds doable, hopefully with nothing more than a parameterized FROM line in the System dockerfiles.

**Comment by gabor-boros** [comment]
If I remember correctly, the `FROM` cannot be parameterized, however, it seems that there were some docker issues and now I could build for those ubuntu images that are supported by s390x and were failing to pull before (`jammy`, `kinetic`)

Also, I found https://hub.docker.com/r/s390x/rockylinux , so we could build `rocky9`. For debian we could also build `stretch` and `buster by using https://hub.docker.com/r/s390x/debian.

Although we _could_ build these, I'm not sure if we _want_ to build it. It is already a big effort to build for supported distros in 3 architectures. At the moment, we are building packages for 15 distros * 2 (ARM, AMD) + 8 currently building s390x + windows + mac = 40 packages. On the top of that, we have the homebrew formulae and dockerfiles to maintain during a release.

**Comment by srh** [comment]
You're right, it seems the main FROM argument can't be parameterized, only the tag can be.
